delivery_postlogistics/models/stock_picking.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import base64
+from operator import attrgetter
+
+import lxml.html
+
+from odoo import _, api, exceptions, fields, models
+
+from ..postlogistics.web_service import PostlogisticsWebService
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    delivery_fixed_date = fields.Date(
+        "Fixed delivery date", help="Specific delivery date (ZAW3217)"
+    )
+
+    # TODO: consider refactoring these fields using a partner relation instead
```

Comment:
could this migration be the opportunity to take care of this?

Reply:
This migration MUST be the opportunity to do so and will. I agreed.